### DISCUSSION TITLE
Types decorated with `ExpandableObjectConverter` are serialized as strings

### DISCUSSION THREAD

**Discussion by lawrence-laz** [VerifyTests/Verify#1036] [Q&A]
Hi,

I have a type:
```csharp
[TypeConverter(typeof(ExpandableObjectConverter))]
public class Foo
{
    public int Id { get; set; }
    public int Bar { get; set; }
    public int Baz { get; set; }
    
    public override string ToString() => $"Foo {Id}";
}
```
and it seems like the attribute with `ExpandableObjectConverter` makes Verify serialize objects of this type as a strings "Foo 1", "Foo 2", etc. instead of as a complex object. Because of this other properties are not persisted in the verified file.

How can I disable this behaviour in Verify?

**Comment by SimonCropp** [comment]
what is the use case for overriding the ToString?

**Reply by lawrence-laz** [reply]
The object is being displayed in [WinForms PropertyGrid](https://learn.microsoft.com/en-us/dotnet/api/system.windows.forms.propertygrid?view=netframework-4.7.2), while the object is collapsed, only `.ToString()` result is visible on that line, when it's expanded, each property is shown on a separate row.

**Reply by lawrence-laz** [reply]
I take it there is no way to disable this right now?

**Reply by SimonCropp** [reply]
can you share a failing unit test as a pull request
